modules/web-console-query-cli.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * web_console/web-console-query-cli.adoc
+
+[id="web-console-query-cli_{context}"]
+= Query image vulnerabilities from the CLI
+Using the `oc` command, you can display information about
+vulnerabilities detected by the Container Security Operator.
+
+.Prerequisites
+* Be running the Container Security Operator on your
+{product-title} instance
+
+.Procedure
+To query for detected container image vulnerabilities, type:
```

Comment:
Procedure steps need to be in some form of bullets. If you don't want these two steps to be an ordered list, then they should be unordered bullets at least.

Reply:
Also suggest "enter the following command" instead of just "type"

Reply:
That would be okay. They don't have to be in order.

Reply:
Done.